pkg/scheduler/manager.go

```diff
@@ -67,36 +68,59 @@ func (mgr *PodGroupManager) PreFilter(ctx context.Context, pod *corev1.Pod) erro
 }
 
 // Permit permits a pod to run
-func (mgr *PodGroupManager) Permit(ctx context.Context, pod *corev1.Pod, nodeName string) (bool, *apiv1.PodGroup, error) {
-	podGroup, err := mgr.PodGroup(pod)
+func (mgr *PodGroupManager) Permit(ctx context.Context, pod *corev1.Pod, nodeName string) (bool, *apiv1.PodGroup, *apiv1.PodGroupSpec, error) {
+	super, sub, err := mgr.PodGroups(pod)
```

Comment:
I prefer the names `pg` and `pgSpec` to `super` and `sub`

Reply:
The `pgSpec` is not precise, as it's not the spec part of `pg`. How about `podGroup` and `subPodGroup`?

Reply:
Fixed in commit#51f4148